joomla/joomla-cms
# [4.3][Guided Tours] Change URL label

Partial fix for Issue #40020, #40021 .

### Summary of Changes
Change `URL` label to `Relative URL` to indicate relative and not absolute URL without having to read the inline help.


### Testing Instructions
Edit a tour.
See URL label.
Edit a step.
Change Type to Redirect.
See URL label.
